src/jupyter-template/devcontainer-template.json

```diff
@@ -0,0 +1,33 @@
+{
+  "id": "jupyter",
+  "version": "0.0.2",
+  "name": "Jupyter",
+  "description": "A Template to run Jupyter lab",
+  "documentationURL": "https://github.com/verily-src/workbench-app-devcontainers/tree/master/src/jupyter-template",
+  "licenseURL": "https://github.com/verily-src/workbench-app-devcontainers/blob/master/LICENSE",
+  "options": {
+    "cloud": {
+      "type": "string",
+      "description": "VM cloud environment",
+      "proposals": ["gcp", "aws"],
+      "default": "gcp"
+    },
+    "login": {
+      "type": "string",
+      "description": "Whether to log in to workbench CLI",
+      "proposals": ["true", "false"],
+      "default": "false"
+    },
+    "containerImage": {
+      "type": "string",
+      "description": "The container image to use",
+      "default": "debian:bullseye"
+    },
+    "containerPort": {
```

Comment:
is this changing depending on the image?

Reply:
dont' quote me but this is what we did for app proxy stuff

Reply:
Yeah, we currently have a port field in app attributes which we use to set the port for the proxy agent https://github.com/verily-src/verily1/pull/40902/files#diff-88d46f9da504f046fab382aae3a9feb1f9c5ee6676faff4fafff281eef543aa3R48, this change is just to pipe that into the container port too.

Reply:
sorry i meant to respond to the other thread about copyright...